Doc/library/urllib.parse.rst

```diff
@@ -158,6 +158,10 @@ or on combining URL components into a URL string.
    percent-encoded sequences into Unicode characters, as accepted by the
    :meth:`bytes.decode` method.
 
+   The optional argument *max_num_fields* is a flag for the maximum number of
```

Comment:
I would just say "The optional argument *max_num_fields* is the maximum number of ..."

Note: wait until this PR is merged before updating the 3 other PRs :-)